Packs/Traceable/Integrations/Traceable/Traceable.py

```diff
@@ -82,9 +82,6 @@
       userIdentifier: attribute(expression: { key: "userIdentifier" })
       sessionId: attribute(expression: { key: "sessionId" })
       ipAddress: attribute(expression: { key: "ipAddress" })
-      userCountry: attribute(expression: { key: "userCountry" })
-      userCity: attribute(expression: { key: "userCity" })
-      userRoles: attribute(expression: { key: "userRoles" })
       statusCode: attribute(expression: { key: "statusCode" })
```

Comment:
There is a bug with the product which is causing the query to fail if these attributes are included. We have reported this internally however in the mean time removing these so that the query works correctly.